python/BUILD

```diff
@@ -14,3 +14,16 @@
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])  # Apache 2.0
+
+# Python2 is a dependency of rules_docker due to its dependency on
+# containerregistry. To run builds on environments in which 'which python'
```

Comment:
Or if the user has set `python_path` to be something else in their bazelrc.

Reply:
I'm not sure how this would work if the user has set python_path to something else as i think  this flag would override it.

Reply:
I meant just to change the documentation. This runtime will override the python_path flag, which by default is set to "python", but if users have set it to something else, then that will also be overridden.